src/vsr/client.zig

```diff
@@ -451,6 +452,10 @@ pub fn ClientType(
                     pong.header.view,
                 });
                 self.view = pong.header.view;
+                // Even if there is a requst in flight, don't try to retransmit it immediately after
+                // a view change. Instead, ride the on_request_timeout normally  to reduce the size
+                // of thundering heard.
```

Comment:
```suggestion
                // Even if there is a request in flight, don't try to retransmit it immediately after
                // a view change. Instead, ride the on_request_timeout normally to reduce the size
                // of thundering herd.
```

Reply:
Ahh, heard isn't a typo, I genuinely thought that's how you spell it, thanks!